psi4/src/psi4/lib3index/dfhelper.cc

```diff
@@ -212,7 +218,7 @@ void DFHelper::AO_core() {
         // if do_wK added to code, the following will need to be changed to match
         required_core_size_ = naux_ * nbf_ * nbf_;
     } else {
-        // total size of sparse AOs
+        // total size of sparse AOs. beautiful
```

Comment:
?

Reply:
I was really happy to find the calculation so easy to work with.

Reply:
a la `libfock/DiskDFJK.cc: line 1070`